security/permissions.rst

```diff
@@ -54,6 +54,9 @@ Filter Logic Examples:
 Policy Creation Example:
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
+In the example below, we will create an ``AccessPolicyDefinition`` that allows users to read Patients and Observations, and a corresponding ``AccessPolicy`` that applies this definition to two Practitioners. With this access policy, the practitioners Alice and Bob will be able to read Patients and Observations, but they will have no access of any kind to other resources. Should they try to read or search for any other resource, they will receive a 403 Forbidden response once the access policy is applied.
```

Comment:
Can you add a row to the filter examples above where one additional scope is requested by the client and then filtered out by the access policy?